packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -222,9 +222,13 @@ export const makeScheduler = async (
       liveSchedule.clockStep,
       Far('PriceStepWaker', {
         wake(time) {
-          setTimeMonotonically(time);
-          trace('wake step', now);
-          void clockTick(liveSchedule);
+          try {
+            setTimeMonotonically(time);
+            trace('wake step', now);
+            clockTick(liveSchedule);
+          } catch (e) {
+            console.warn(`🧯Auction threw ${e}. Caught in PriceStepWaker.`);
```

Comment:
this is a new emoji to log. what does it mean differently than 🚨 and ⚠️ ? 

Also shouldn't this log as an error?

Reply:
Those are both warnings that something is wrong and danger may persist. This was an attempt to say that the emergency has been handled.

Reply:
lol I get it now.

Is it a full recovery? No further investigation needed?

Reply:
Investigation would be warranted. I'd like to know that something happened and figure out why, but I don't expect the chain to be in imminent danger.

Reply:
Then it sounds like the same attention needed as https://github.com/Agoric/agoric-sdk/blob/33566753adc9803474122e5b52930e7e743a332c/packages/inter-protocol/src/vaultFactory/vaultManager.js#L749-L753

I think it needs to a) go to the error logs (so console.error) and b) indicate that investigation is warranted. ⚠️ is what we've been using for that.

I see how 🧯 might be a better pictograph for that because it suggests an exception was handled, but a counter-argument is that every exception that doesn't halt the vat is handled so it's not an informative distinction.

Reply:
done